protos/camera_server/camera_server.proto

```diff
@@ -58,6 +85,55 @@ message StopVideoResponse {
     int32 stream_id = 1;    // video stream id
 }
 
+message SubscribeStartVideoStreamingRequest {}
+message StartVideoStreamingResponse {
+    int32 stream_id = 1;    // video stream id
+}
+
+message SubscribeStopVideoStreamingRequest {}
+message StopVideoStreamingResponse {
+    int32 stream_id = 1;    // video stream id
+}
+
+message SubscribeSetModeRequest {}
+message SetModeResponse {
+    Mode mode = 1;
+}
+
+message SubscribeStorageInformationRequest {}
+message StorageInformationResponse {
+    int32 storage_id = 1;
+}
```

Comment:
Shouldn't this contain the storage information?

E.g. if you look at the action server, this is what the response contains:
https://github.com/mavlink/MAVSDK-Proto/blob/8966cf59029ac17d81654071ca6d3d27f0d8897d/protos/action_server/action_server.proto#L79-L82